backend/app/Savor22b.Tests/Action/PlantingSeedActionTests.cs

```diff
@@ -0,0 +1,186 @@
+namespace Savor22b.Tests.Action;
+
+using System;
+using Libplanet.State;
+using Xunit;
+using Savor22b.Action;
+using Savor22b.States;
+using Savor22b.Action.Exceptions;
+using System.Collections.Immutable;
+
+public class PlantingSeedActionTests : ActionTests
+{
+
+    public PlantingSeedActionTests()
+    {
+    }
+
+    private InventoryState getInventoryState()
+    {
+        InventoryState inventoryState = new InventoryState();
+        var newSeed = new SeedState(Guid.NewGuid(), 1);
+        inventoryState = inventoryState.AddSeed(newSeed);
+        return inventoryState;
+    }
+
+    private VillageState getVillageState()
+    {
+        VillageState villageState = new VillageState(
+            new HouseState(
+                1,
+                1,
+                1,
+                new HouseInnerState()
+            )
+        );
+        return villageState;
+    }
+
+    private RootState getRootState()
```

Comment:
```suggestion
    private RootState createRootStatePreset()
```
createRootStatePreset 같은 이름이 더 좋지 않을까요